bot/modules/speech.js

```diff
@@ -243,7 +244,7 @@ request.post(
 };
 
 function inPrivateOrBotSandbox(msg){
-  if((msg.channel.type == 'dm') || (msg.channel.id === ChannelID)){
+  if((msg.channel.type == 'dm') || (msg.channel.id === ChannelID1) || (msg.channel.id === ChannelID2) || (msg.channel.id === ChannelID3) || (msg.channel.id === ChannelID4) || (msg.channel.id === ChannelID5) || (msg.channel.id === ChannelID6)){
```

Comment:
use the config array and then array.includes(msg.channel.id) instead  of all of these or statements :)